libraries/imgui-opencv-poco/msvc/msvc.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
What about keep only Cmake? If you keep msvc solution, it will one more file to maintain.

Reply:
This is for the new blog post, that uses pure MSVC